factory/glideFactoryConfig.py

```diff
@@ -99,7 +99,7 @@ def __init__(self, entry_name, config_file, convert_function=repr):
 # and join the results
 class JoinConfigFile(ConfigFile):
     def __init__(self, entry_name, config_file, convert_function=repr):
-        ConfigFile.__init__(self, config_file, convert_function)
+        ConfigFile.__init__(self, os.path.join("entry_" + entry_name, config_file), convert_function)
```

Comment:
This is wrong. It makes the results ignore the common values.

Reply:
I think you are right, I misunderstood how the code works.